### DISCUSSION TITLE
Adress with path

### DISCUSSION THREAD

**Discussion by prizzledev** [coder/code-server#3255] [Q&A] [answered]
Hey, i want to run code-server with a path behind my address, for examlpe: 192.168.0.67:8080/PATH is that possible?
Because I want to config that if I type in the Browser 192.168.0.67:8080 i want to display my own .html file that contains a link to 192.168.0.67:8080/PATH where code-server is displayed. 

Thank you for your help<3

**Comment by code-asher** [comment, marked as answer]
You'll need a reverse proxy (like Caddy or NGINX) to put code-server behind a path.

**Reply by code-asher** [reply]
There is an example in the docs for using a sub-path with Caddy: https://github.com/cdr/code-server/blob/main/docs/guide.md#3-expose-code-server
